apel/db/backends/mysql.py

```diff
@@ -404,6 +404,59 @@ def summarise_cloud(self):
                 self.db.rollback()
             raise
 
+    def clean_stale_summaries(self, db_type, summariser_start_time, threshold):
+        """For the DB type, remove stale summaries newer than the threshold."""
+        if db_type == "cloud":
+            self._clean_stale_cloud_summaries(summariser_start_time, threshold)
+        else:
+            raise ValueError("Attempting to clean up summaries in a %s db, "
```

Comment:
Why don't we just ignore the setting for non-cloud schemas, and document what it currently works on? Maybe log a warning.

Reply:
I think I've addressed this in [9960f97](https://github.com/apel/apel/pull/222/commits/9960f97d4d4298b62abddcc526ce1a62adbe95db), but again - not yet tested I've not broken the functionality.

Reply:
Added comment to config file to clarify that this is Cloud only,